engine/schema/src/main/resources/META-INF/db/schema-41720to41800.sql

```diff
@@ -1569,4 +1569,20 @@ CREATE VIEW `cloud`.`user_view` AS
             left join
         `cloud`.`async_job` ON async_job.instance_id = user.id
             and async_job.instance_type = 'User'
-            and async_job.job_status = 0;
\ No newline at end of file
+            and async_job.job_status = 0;
+
+--- Create table for Virtual Machine Schedule Entity
+
+CREATE TABLE IF NOT EXISTS `cloud`.`vm_schedule` (
```

Comment:
note - after 4.18 is released, this would need to be moved to a new sql upgrade path (4.18->4.19)

Reply:
Noted

Reply:
Now is the time :)

Reply:
@nvazquez I did not understand.

Reply:
To move the DB update code to the 4.18 -> 4.19 update script, @rahulbcn27 !